tools/monitoring/alert/alert.go

```diff
@@ -58,8 +58,8 @@ func (c *Client) RunAlerting() {
 }
 
 func (c *Client) handleReportMessage(rmsg *prowapi.ReportMessage) {
-	if rmsg.Status == prowapi.SuccessState || rmsg.Status == prowapi.FailureState || rmsg.Status == prowapi.AbortedState {
-		log.Printf("Received Pubsub message %v\n", rmsg)
+	if rmsg.Status == prowapi.FailureState || rmsg.Status == prowapi.AbortedState {
+		log.Printf("Received Pubsub message in %s state: %v\n", rmsg.Status, rmsg)
```

Comment:
FYI, dumping rmsg also includes the states as well. But this is ok if you'd like to be more explicit on the state in the log.
```
Received Pubsub message &{knative-tests knative-monitoring ci-knative-client-continuous success https://prow.knative.dev/view/gcs/knative-prow/logs/ci-knative-client-continuous/1188909808967749632 gs://knative-prow/logs/ci-knative-client-continuous/1188909808967749632 [{knative client  master   [] knative.dev/client  false 0}] periodic ci-knative-client-continuous}
```

Reply:
Let's be more explicit so that we can filter message with what we defined here